admin/views/administrator/administration-home.ejs

```diff
@@ -37,12 +37,16 @@
         </div>
     </div>
 
-    <div class="column-one-third sidebar-top-border" tabindex="0" role="group" aria-label="sidebar">
-        <h3 class="heading-medium" tabindex="0">Before you start</h3>
-        <ul class="list">
-            <li>
-                <a href="/PDFs/MTC_administration_guidance_June-2017-trial.pdf" target="_blank">Guidance</a>
-            </li>
-        </ul>
+    <div class="column-one-third add-title-margin" tabindex="0" aria-label="sidebar">
+        <aside role="complementary" class="support-column">
+            <h3 class="heading-medium" tabindex="0">Before you start</h3>
+            <nav role="navigation">
+                <ul class="list">
+                    <li>
+                        <a href="/PDFs/MTC_administration_guidance_June-2017-trial.pdf" target="_blank">Guidance</a>
```

Comment:
I am not a big fan of exposing our directories in the frontend.
How about we handle this with a router get method?
Same as my PR.

Reply:
@GeorgeChatzigiannis I think this is a topic for a different discussion.  We may not even want Express to serve static assets.